httpservices/src/main/java/ucar/httpservices/HTTPMethod.java

```diff
@@ -349,12 +349,10 @@ public HttpResponse executeRaw() throws HTTPException {
       setcontent(rb);
       setheaders(rb, this.headers);
       this.lastrequest = buildRequest(rb, this.settings);
-      AuthScope methodscope = HTTPAuthUtil.uriToAuthScope(this.methodurl);
-      AuthScope target = HTTPAuthUtil.authscopeUpgrade(session.getSessionScope(), methodscope);
       // AFAIK, targethost, httpclient, rb, and session
       // contain non-overlapping info => we cannot derive one
       // from any of the others.
-      HttpHost targethost = HTTPAuthUtil.authscopeToHost(target);
+      HttpHost targethost = new HttpHost(this.methodurl.getHost());
```

Comment:
Use the 3-parameter constructor to resolve issue #140

Reply:
Yes. Using HttpHost targethost = new HttpHost(this.methodurl.getHost(), this.methodurl.getPort(), this.methodurl.getScheme()); worked for me.